tools/internal_ci/macos/grpc_master.cfg

```diff
@@ -15,11 +15,16 @@
 # Config file for the internal CI (in protobuf text format)
 
 # Location of the continuous shell script in repository.
-build_file: "grpc/tools/internal_ci/macos/grpc_master.sh"
+build_file: "grpc/tools/internal_ci/macos/grpc_run_tests_matrix.sh"
 timeout_mins: 240
 action {
   define_artifacts {
     regex: "**/*sponge_log.xml"
     regex: "github/grpc/reports/**"
   }
 }
+
+env_vars {
+  key: "RUN_TESTS_FLAGS"
+  value: "-f basictests macos --internal_ci -j 2 --inner_jobs 4"
```

Comment:
The Macs aren't configured to have access to our BQ results, but the Windows might have access already.